MANAGE/package.json

```diff
@@ -6,9 +6,6 @@
   "scripts": {
     "create": "migrate-mongo create",
     "down": "migrate-mongo down",
-    "postinstall": "pipenv install --skip-lock",
```

Comment:
Is it typically better practice for python apps to avoid tools like pipenv?

It seems like Docker does the same sorts of things.  Am I correct in understanding that it's a proper replacement?

Reply:
that line should be removed, it was there from the pre-docker branch that relied on pipenv. In fact, the node.js manage sub-project does not directly interact with flask web sub-project

Reply:
Docker is replacing pipenv, yes. The intial build of a docker container creates a sandboxed immutable python application. This means that the image must be rebuilt if any dependencies are added. But documentation of all dependencies is implicit in the development process.

Reply:
This is more of a general question about dependency management tools for python.  Is it a more common practice to manage dependencies with a docker task rather than with a separate tool like pipenv?

Reply:
I don’t honestly know, my experience with Python is mostly personal. I found early on, that docker gave me more consistency,  made version management easier, and made removing dependencies simple.

Reply:
pipenv does the double edged sword of not redownloading all dependencies whenever one is added. Though this can lead to a bloated dependency folder if improperly managed.